tests/features/support-request.feature

```diff
@@ -0,0 +1,27 @@
+Feature: AMP Support Request
```

Comment:
It would be great to also test data for a validated URL is in the output as well.